src/helm/proxy/clients/critique_client.py

```diff
@@ -240,64 +243,154 @@ class ScaleCritiqueClient(CritiqueClient):
     - A `CritiqueResponse` maps to TODO
     """
 
-    def __init__(self, api_key: str, cache_config: CacheConfig):
+    def __init__(
+        self,
+        api_key: str,
+        cache_config: CacheConfig,
+        project_name: Optional[str] = None,
+        batch_name: Optional[str] = None,
+    ):
         self._cache = Cache(cache_config)
-        self.client = scaleapi.ScaleClient(api_key)
+        self._client = scaleapi.ScaleClient(api_key)
+        self._project_name = project_name
+        self._batch_name = batch_name
+
+    def _get_or_create_scale_project_and_batch(self, template: CritiqueTaskTemplate) -> Tuple[str, str]:
```

Comment:
There are many problems with this method; in the interest of time, I would suggest just deleting it and using the user-specified batch name.